src/plugins/registration.go

```diff
@@ -29,6 +29,7 @@ const (
 	dataplaneSoftwareDetailsMaxWaitTime = time.Duration(5 * time.Second)
 	// Time between attempts to gather DataplaneSoftwareDetails
 	softwareDetailsOperationInterval = time.Duration(1 * time.Second)
+	nginxStartMaxWaitTime            = 0
```

Comment:
Would this not essentially turn off backoff altogether, if the `MaxElapsedTime` is zero? If that is intentional, then I would maybe adding a comment explaining why it is turned off.

Reply:
When the max wait time is set to 0 backoff will continue waiting indefinitely until the master process is found. I'll add in a comment now